src/runtime/ScriptAsyncFunctionObject.h

```diff
@@ -63,21 +63,8 @@ class ScriptAsyncFunctionObject : public ScriptFunctionObject {
     Object* m_homeObject;
 };
 
-class ScriptAsyncFunctionHelperFunctionObject : public NativeFunctionObject {
-public:
-    ScriptAsyncFunctionHelperFunctionObject(ExecutionState& state, NativeFunctionInfo info, ExecutionPauser* executionPauser, ScriptAsyncFunctionObject* sourceFunction)
-        : NativeFunctionObject(state, info)
-        , m_executionPauser(executionPauser)
-        , m_sourceFunction(sourceFunction)
-    {
-    }
-
-    ExecutionPauser* m_executionPauser;
-    ScriptAsyncFunctionObject* m_sourceFunction;
-};
-
-Value awaitFulfilledFunction(ExecutionState& state, Value thisValue, size_t argc, Value* argv, bool isNewExpression);
-Value awaitRejectedFunction(ExecutionState& state, Value thisValue, size_t argc, Value* argv, bool isNewExpression);
```

Comment:
These two global functions seem to be invoked only inside the new `await` function. How about defining the two functions as static function or member function of `ScriptAsyncFunctionObject`?

Reply:
I updated it